packages/core/modules-sdk/src/medusa-module.ts

```diff
@@ -576,4 +576,6 @@ export class MedusaModule {
 }
 
 global.MedusaModule ??= MedusaModule
-exports.MedusaModule = global.MedusaModule
+const GlobalMedusaModule = global.MedusaModule as typeof MedusaModule
+
+export { GlobalMedusaModule as MedusaModule }
```

Comment:
@carlos-r-l-rodrigues can you check if this is correct.

Reply:
should work the same 👍